### PR TITLE
Prepare release v1.39.0

### PR DESCRIPTION
**Pull request by pavolloffay**
Depends on https://github.com/jaegertracing/jaeger-ui/pull/1010

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,48 @@ next release
 
 ### UI Changes
 
+1.39.0 (2022-10-31)
+-------------------
+### Backend Changes
+
+#### ⛔ Breaking Changes
```
Comment (yurishkuro) [inline CHANGELOG.md:20]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,48 @@ next release
...
+
+#### New Features
+
+* Add support for OpenSearch 2.x ([@gaurav-05](https://github.com/gaurav-05) in [#3966](https://github.com/jaegertracing/jaeger/pull/3966))
+
+#### Bug fixes, Minor Improvements
+
+* Pin SBOM action to commit ([@yurishkuro](https://github.com/yurishkuro) in [bb49249](https://github.com/jaegertracing/jaeger/commit/bb492490594c9d9321ed9242862ac2a8864ff771))
+* Bump github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger ([@dependabot[bot]](https://github.com/apps/dependabot) in [#4009](https://github.com/jaegertracing/jaeger/pull/4009))
```
Comment (yurishkuro) [inline CHANGELOG.md:29]: Should we be listing all these dep bumps? I think they are just noise for release notes

Comment (pavolloffay) [reply CHANGELOG.md]: I have removed all dependabot PRs.

File: CHANGELOG.md [hunk 3]
```diff
@@ -13,6 +13,32 @@ next release
...
+### Backend Changes
+
+#### New Features
+
+* Add support for OpenSearch 2.x ([@gaurav-05](https://github.com/gaurav-05) in [#3966](https://github.com/jaegertracing/jaeger/pull/3966))
+
+#### Bug fixes, Minor Improvements
+
+* Pin SBOM action to commit ([@yurishkuro](https://github.com/yurishkuro) in [bb49249](https://github.com/jaegertracing/jaeger/commit/bb492490594c9d9321ed9242862ac2a8864ff771))
+* don't require auth ([@joe-elliott](https://github.com/joe-elliott) in [#4005](https://github.com/jaegertracing/jaeger/pull/4005))
```
Comment (albertteoh) [inline CHANGELOG.md:27]:
```suggestion
* Remove auth requirement on monitor demo ([@joe-elliott](https://github.com/joe-elliott) in [#4005](https://github.com/jaegertracing/jaeger/pull/4005))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -13,6 +13,32 @@ next release
...
+* Adding anchore for SBOM signing during release ([@jkowall](https://github.com/jkowall) in [#3987](https://github.com/jaegertracing/jaeger/pull/3987))
+* Bump sarama to 1.33.0 ([@pavolloffay](https://github.com/pavolloffay) in [#3983](https://github.com/jaegertracing/jaeger/pull/3983))
+* Add note on jaeger grpc storage compliance ([@arajkumar](https://github.com/arajkumar) in [#3985](https://github.com/jaegertracing/jaeger/pull/3985))
+* Added link to FOSSA and Artifact Hub to README ([@jkowall](https://github.com/jkowall) in [#3964](https://github.com/jaegertracing/jaeger/pull/3964))
+* Add grafana container to monitor docker-compose ([@albertteoh](https://github.com/albertteoh) in [#3955](https://github.com/jaegertracing/jaeger/pull/3955))
+* Expose storage integration helpers as go pkg ([@arajkumar](https://github.com/arajkumar) in [#3944](https://github.com/jaegertracing/jaeger/pull/3944))
+
+### UI 
+
+* UI pinned to version [1.27.2](https://github.com/jaegertracing/jaeger-ui/blob/main/CHANGELOG.md#v1272-2022-11-01) to bump dependencies.
```
Comment (albertteoh) [inline CHANGELOG.md:40]:
```suggestion
* UI pinned to version [1.27.2](https://github.com/jaegertracing/jaeger-ui/blob/main/CHANGELOG.md#v1272-2022-11-01).
```

File: RELEASE.md [hunk 5]
```diff
@@ -52,7 +52,7 @@ Here are the release managers for future versions with the tentative release dat
 
 | Version | Release Manager | Tentative release date |
 |---------|-----------------|------------------------|
-| 1.39.0  | @albertteoh     | 2 November  2022       |
 | 1.40.0  | @joe-elliott    | 7 December  2022       |
-| 1.41.0  | @pavolloffay    | 4 January   2023       |
+| 1.41.0  | @albertteoh     | 4 January   2023       |
```
Comment (albertteoh) [inline RELEASE.md:56]: Thanks for updating this.

**Review by albertteoh — APPROVED** [review]
LGTM

**Pull request merged**